src/python/pants/backend/python/tasks/resolve_requirements_task_base.py

```diff
@@ -34,6 +35,12 @@ def prepare(cls, options, round_manager):
     round_manager.require_data(PythonInterpreter)
     round_manager.optional_product(PythonRequirementLibrary)  # For local dists.
 
+  @staticmethod
```

Comment:
This method is not static because it uses `self`.

This code needs a unit or integration test.